lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -535,6 +538,26 @@ def _delete_environment(self, environment: Environment, log: Logger) -> None:
             else:
                 log.debug("not wait deleting")
 
+    def _save_console_log(
+        self, resource_group_name: str, environment: Environment, log: Logger
+    ) -> None:
+        compute_client = get_compute_client(self)
+        vms = compute_client.virtual_machines.list(resource_group_name)
+        saved_path = environment.log_path / get_datetime_path()
```

Comment:
this is clearer to know, before entering the folder, `environment.log_path / f“{get_datetime_path()}_serial_log"`

Reply:
updated